hack/verify-flags/exceptions.txt

```diff
@@ -1,4 +1,87 @@
 Vagrantfile:      node_ip = $node_ips[n]
+_output/release-stage/full/kubernetes/Vagrantfile:      node_ip = $node_ips[n]
```

Comment:
Thanks @wojtek-t for catching this. As I wrote here https://github.com/kubernetes/kubernetes/pull/26572#discussion_r65225728 I have no idea what does it mean. Shouldn't this be caught by verify-all.sh script?